devices/Ssd13xx/samples/Program.cs

```diff
@@ -13,14 +13,15 @@
 //////////////////////////////////////////////////////////////////////
 // when connecting to an ESP32 device, need to configure the I2C GPIOs
 // used for the bus, for example
-Configuration.SetPinFunction(21, DeviceFunction.I2C1_DATA);
-Configuration.SetPinFunction(22, DeviceFunction.I2C1_CLOCK);
+Configuration.SetPinFunction(22, DeviceFunction.I2C1_DATA);
+Configuration.SetPinFunction(20, DeviceFunction.I2C1_CLOCK);
 //////////////////////////////////////////////////////////////////////
 
 //Tested with 128x64 and 128x32 OLEDs
-using Ssd1306 device = new Ssd1306(I2cDevice.Create(new I2cConnectionSettings(1, Ssd1306.DefaultI2cAddress)), Ssd13xx.DisplayResolution.OLED128x64);
+using Ssd1306 device = new Ssd1306(I2cDevice.Create(new I2cConnectionSettings(1, Ssd1306.DefaultI2cAddress)), Ssd13xx.DisplayResolution.OLED64x128);
```

Comment:
Rather than changing this line, add another line and add a comment.

Reply:
Weird.  I thought I'd left the default one there.  My test changes must have made it in.  I don't think I really need to show the other option in a comment.  I'll just make sure it's still the original line.